example/isolate-service/README.md

```diff
@@ -0,0 +1,50 @@
+# Description
```

Comment:
This seems to be copied verbatim from the hello world folder and needs some updates, e.g. you can explain that this is an example of load balancing using multiple isolates listening to the same socket which is made possible by passing `shared: true` to `Server.serve`. 

I would also rename the folder from `isolate-service` to `helloworld-with-isolates`.